openedx/core/djangoapps/persistenttask/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models to support persistent tasks.
+"""
+
+from django.db import models
+from jsonfield import JSONField
+
+
+class FailedTask(models.Model):
+    """
+    Representation of tasks that have failed
+    """
+    task_name = models.CharField(max_length=255)
+    task_id = models.CharField(max_length=255, db_index=True)
+    args = JSONField(blank=True)
+    kwargs = JSONField(blank=True)
+    exc = models.CharField(max_length=255)
+    datetime_failed = models.DateTimeField(db_index=True)
+    datetime_resolved = models.DateTimeField(blank=True, null=True, default=None)
+
+    class Meta(object):
+        index_together = [
+            (u'task_name', u'exc'),
```

Comment:
Thanks @jcdyer .  Do you have an output of an example table from running the code manually?  I'm curious to know what the value of the `exc` field can be - if they are always different, does that make the index costly?